starter-code/src/App.js

```diff
@@ -1,19 +1,61 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import FoodBox from './components/FoodBox'
+import AddButton from './components/AddButton';
+import SearchBar from './components/SearchBar';
+import MenuCount from './components/MenuCount';
+import Foods from './foods.json'
 import './App.css';
+import 'bulma/css/bulma.css';
 
 class App extends Component {
+
+  state = {
+    foodFilter: "",
+    listFood: Foods,
+    countFood: ""
+  }
+
+  addFoodHandler = (theFood) => {
+    const newListFood = [...this.state.listFood];
+    newListFood.push(theFood);
```

Comment:
esto q haces es una tontería, I mean, la mitad bien, y la otra mitad muy mal... 🤓
Me refiero a que ya que utilizas ES6 para crear un array nuevo con todos los items de `this.state.listFood`, sólo con poner una coma y añadir el item que quieres "pushear", ya lo tienes.

`const newListFood = [ ...this.state.listFood, theFood ]`

😎